stock_quant_history/models/stock_quant_history_snapshot.py

```diff
@@ -122,63 +112,154 @@ def _generate_stock_quant_history(self):
                 }
             )
         )
-        self.previous_snapshot_id = previous_quant_snapshot
-
-        _logger.info("Processing %s from %s", self.name, self.previous_snapshot_id.name)
-        if previous_quant_snapshot.stock_quant_history_ids.exists():
+        duplicated_fields = ", ".join(self.env["stock.quant.history"]._fields_to_copy())
```

Comment:
nitpick: I would suggest renaming this to `fields_to_duplicate`, because on first read I instantly though that this variable stored fields that where duplicated, but it just stores fields that WILL BE duplicated.

Reply:
a follow-up is here: https://github.com/OCA/stock-logistics-reporting/pull/430